Cargo.toml

```diff
@@ -32,3 +32,4 @@ cupid = "0.3"
 
 [features]
 strict = []
+std = []
```

Comment:
Should there be `default = ["std"]`?

Reply:
I thought about this, and I think we should settle it before the next release. I've opened #193.